docs/course_team/concepts/learner_groups_dashboard.rst

```diff
@@ -0,0 +1,35 @@
+At-Risk Learner Dashboard Reports
```

Comment:
This one is linked to: https://docs.openedx.org/projects/openedx-aspects/page/reference/learner_groups_dashboard.html

Reply:
@bmtcril, I understand. I moved these files in the following PR.